_data/simple-icons.json

```diff
@@ -3940,6 +3940,11 @@
             "source": "https://www.hackster.io/branding#logos",
             "guidelines": "https://www.hackster.io/branding"
         },
+        {
+            "title": "Handlebars.js",
+            "hex": "000000",
+            "source": "https://raw.githubusercontent.com/handlebars-lang/docs/master/src/.vuepress/public/icons/handlebarsjs-icon.svg"
```

Comment:
@jorgeamadosoria The links would need to be updated to:
https://github.com/handlebars-lang/docs/blob/2060872f4774f8fa636bbf837cfd4dac0996b339/src/.vuepress/public/icons/handlebarsjs-icon.svg

Reply:
changed the source in #6422